src/cms/templates/events/_event_filter_form.html

```diff
@@ -0,0 +1,94 @@
+                <div class="w-1/2 pr-1">
+                    <label for="before_date" class="mb-2 block font-bold text-sm">{% trans 'Not after' %}</label>
+                    {% render_field filter_form.before_date|add_error_class:"border-red-500" class+="appearance-none block w-full bg-gray-200 text-gray-600 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-400" %}
+                </div>
+                <div class="w-1/2 pl-1">
+                    {% render_field filter_form.before_time|add_error_class:"border-red-500" class+="appearance-none block w-full bg-gray-200 text-gray-600 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-400" %}
+                </div>
+            </div>
+        </div>
+        <div class="w-1/3 pr-6">
+            <h3 class="font-bold mb-3">{% trans 'Location' %}</h3>
+            <div class="relative my-2">
+                {% trans 'Search' as search_placeholder %}
+                <input id="poi-query-input" type="search" autocomplete="off" class="no-new-poi pr-8 appearance-none block w-full bg-gray-200 text-gray-600 placeholder-gray-600 border border-gray-200 rounded py-3 px-4 leading-tight focus:outline-none focus:bg-white focus:border-gray-400" placeholder="{% if filter_poi %}{{ filter_poi|poi_translation_title:current_language }}{% else %}{{ search_placeholder }}{% endif %}" data-url="{% url 'search_poi_ajax' %}" data-region-slug="{{ region.slug }}" data-default-placeholder="{{ search_placeholder }}">
+                <div class="absolute inset-y-0 right-0 flex items-center px-2 text-gray-800">
+                    <button id="poi-remove" title="{% trans 'Empty query input' %}">
+                        <i data-feather="trash-2" class="h-5 w-5"></i>
+                    </button>
+                </div>
+            </div>
+            {% render_field filter_form.poi_id id="poi-id" class+="hidden" %}
+            <div class="relative" id="poi-query-result">
+                {% include 'events/_poi_query_result.html' %}
+            </div>
+        </div>
+        <div class="w-1/3">
+            <h3 class="font-bold mb-3">{% trans 'All day events' %}</h3>
+            <div class="flex flex-wrap">
+                {% render_field filter_form.all_day|add_error_class:"border-red-500" class+="leading-normal rounded" %}
+            </div>
+            <h3 class="font-bold mb-3 pt-4">{% trans 'Recurring events' %}</h3>
+            <div class="flex flex-wrap">
+                {% render_field filter_form.recurring|add_error_class:"border-red-500" class+="leading-normal rounded" %}
+            </div>
+        </div>
+    </div>
+    <div class="w-full flex p-4 justify-end">
+        <button type="button" id="filter-reset"
+                class="bg-red-500 hover:bg-red-600 text-white font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline mr-3">
```

Comment:
```suggestion
                class="bg-gray-500 hover:bg-gray-600 text-white font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline mr-3">
```
I think the red color is a bit "drastic" here, I'd suggest something less flashy, e.g. gray.

Reply:
Makes sense, I agree :+1: